src/chain/sanitizers-impl.spec.ts

```diff
@@ -103,3 +104,28 @@ describe('#customSanitizer()', () => {
     });
   });
 });
+
+describe('#toArray()', () => {
+  it('adds toArray() sanitizer to the context', () => {
+    const ret = sanitizers.toArray();
+
+    expect(ret).toBe(chain);
+    expect(context.addSanitization).toHaveBeenCalledWith(expect.any(Function), {
+      custom: true,
+    });
+  });
+
+  it('sanitizes to array', () => {
+    sanitizers.toArray();
+
+    const meta: Meta = { req: {}, location: 'body', path: 'foo' };
+    const toArray = context.addSanitization.mock.calls[0][0] as CustomSanitizer;
+
+    expect(toArray([], meta)).toEqual([]);
+    expect(toArray('foo', meta)).toEqual(['foo']);
+    expect(toArray(['foo'], meta)).toEqual(['foo']);
+    expect(toArray('', meta)).toEqual(['']);
+    expect(toArray(null, meta)).toEqual([null]);
```

Comment:
Do you see falsy values/nulls becoming an issue when they aren't transformed into `[]`?

Reply:
For clarification: I wonder if other users won't ask for a fine-grained control like in `optional`.

But nonetheless, I'm fine with whatever 👍

Reply:
I was wondering the same thing while implementing the test cases and didn't come to a conclusion on that. You're talking about transforming `[4,5,null,'',6]` to `[4,5,6]` by means of specifying some sort of `checkFalsy` and/or `checkNull` modifiers, right? I think we could always add options for that at a later time if the need arises. What do you think?

Reply:
> You're talking about transforming `[4,5,null,'',6]` to `[4,5,6]` by means of specifying some sort of `checkFalsy` and/or `checkNull` modifiers, right?

Not that, but whether `null` should become `[]` (like `undefined`), and then that could evolve even crazier... but oh well.

As I said, this code looks fine to me, so will merge it.